docs/basics/controller.md

```diff
@@ -0,0 +1,83 @@
+# Controller
+
+### Table of Contents
+
+* [What is REST API](#what-is-rest-api)
```

Comment:
I don't think we need to explain REST API in our docs. I'm assuming if the user is looking forward to scaffold rest controllers then he must know what is a REST API